File: my-folder/2264-minimum-sum-of-four-digit-number-after-splitting-digits/solution.java
class Solution {
    public int minimumSum(int num) {
      String s=(num+"");
       char ch[]=s.toCharArray();
        Arrays.sort(ch);
        int a=Integer.parseInt(""+ch[0]+ch[2]);
        int b=Integer.parseInt(""+ch[1]+ch[3]);
       return a+b;
        
        
    }
}

File: my-folder/2264-minimum-sum-of-four-digit-number-after-splitting-digits/solution.cpp
class Solution {
public:
    int minimumSum(int num) {
        vector<int>arr;
        arr.push_back(num%10);
        num/=10;
         arr.push_back(num%10);
        num/=10;
         arr.push_back(num%10);
        num/=10;
         arr.push_back(num%10);
        num/=10;
        sort(arr.begin(),arr.end());
        return (arr[0]*10+arr[2])+(arr[1]*10+arr[3]);
        
    }
};
